src/CLI/Check.php

```diff
@@ -46,6 +50,12 @@ protected function configure(): void
                 't',
                 InputOption::VALUE_OPTIONAL,
                 'Target php version to use for parsing'
+            )
+            ->addOption(
+                self::STOP_ON_FAILURE_PARAM,
+                's',
+                InputOption::VALUE_OPTIONAL,
+                'Stop on failure'
```

Comment:
We can try to add the default value and manage it like in this link,
https://symfony.com/doc/current/console/input.html#options-with-optional-arguments

What do you think @fain182 ? The Symfony example represents our case, the behavior of the optional parameter.

Reply:
I studied it a little, and I think that this is not a VALUE_OPTIONAL, but a VALUE_NONE, because we will not pass any argument... and with VALUE_NONE I can't set the default, but anyway is false, like you correctly thought...

Reply:
Great!